packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6866,6 +6889,8 @@ export default class Meeting extends StatelessWebexPlugin {
     }
 
     if (floorRequestNeeded) {
+      this.screenShareFloorState = ScreenShareFloorStatus.PENDING;
```

Comment:
there is 1 more place where we enqueue SHARE_FLOOR_REQUEST, so we need to set that flag there too (probably best to create a helper function enqueueShareFloorRequest() and also we need to set `this.screenShareFloorState` to `ScreenShareFloorStatus.PENDING` in `requestScreenShareFloor()` itself, because it gets called from `requestScreenShareFloorIfPending()` without the queuing.